microsetta_public_api/api/datasets.py

```diff
@@ -0,0 +1,18 @@
+from microsetta_public_api.utils._utils import jsonify
+from microsetta_public_api.resources_alt import get_resources
+from microsetta_public_api.config import schema
+
+
+def available():
+    resources = get_resources()
+    escape = {schema.metadata_kw}
+    dataset_key = 'datasets'
+
+    if dataset_key not in resources:
+        return jsonify([]), 200
+
+    datasets = list(filter(
+        lambda x: x not in escape,
```

Comment:
If there are any security/privilege/privacy concerns with exposing the data behind schema.metadata_kw, the policy for denoting that it is hidden should be centralized to also check what happens when the user passes in the escaped key(s).  If we're perfectly happy to give them this data when a user passes in schema.metadata_kw, and this is just a facade to make things easier to the user, then the current implementation is totally fine.

Reply:
Just a facade to make things easier for the API user.